applications/osb-portal/src/pages/RepositoryPage.tsx

```diff
@@ -322,7 +329,10 @@ export const RepositoryPage = (props: any) => {
 
 
                     <Typography component="h2" variant="h2" className="primary-heading">
-                      Overview
+                      Overview <Tooltip title={`Repositories provide views of files in public resources that have been indexed in OSBv2 by users. Use the Repository Contents pane on the right to select files from this repository to add to your workspaces.`}>
```

Comment:
In order to vertically align the text the best solution is to use flex. In material using [boxes](https://mui.com/components/box/) help, something like:

```jsx
<Box display="flex" alignItems="center" justifyContent="space-between">
<Typography>...</Typography>
<Tooltip></Tooltip>
</Box>
```

Reply:
Cool, let me go tinker with that.